.ci/travis/travis.sh

```diff
@@ -655,6 +655,62 @@ check-since-version)
   fi
   ;;
 
+checkstyle-cli-run-openjdk14)
+  # Set up environment
+  CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
+                   --non-recursive org.codehaus.mojo:exec-maven-plugin:1.6.0:exec)
+  CHECKSTYLE_DIR=$(pwd)
+  TRAVIS_DIR="${CHECKSTYLE_DIR}/.ci/travis"
+  CHECKSTYLE_CONFIG="${CHECKSTYLE_DIR}/.ci/travis/openjdk14-test/single-module-config.xml"
+  FILTER_FILE="${CHECKSTYLE_DIR}/.ci/travis/openjdk14-test/jdk14-test-excluded-files.list"
+  INPUT_FILE="$(mktemp)"
+  OUTPUT_FILE="$(mktemp)"
+
+  # Build Checkstyle
+  mvn -e -P assembly package
+  CHECKSTYLE_JAR="${CHECKSTYLE_DIR}/target/checkstyle-${CS_POM_VERSION}-all.jar"
+
+  # Set up input file for Checkstyle run
+  echo "Downloading list of openjdk14 test files..."
+  mkdir -p .ci-temp/checkstyle-cli-run-openjdk14
+  cd .ci-temp/checkstyle-cli-run-openjdk14
+  # Below link is our copy of jdk14 test input file list, in case of download.java.net failure
+  # wget https://raw.githubusercontent.com/checkstyle/contribution/master/misc/jdk14-test-files.list
+  wget https://download.java.net/openjdk/testresults/14/archives/36/langtools-36-summary.txt
```

Comment:
...or use it here. Something like `wget https://download.java.net/openjdk/testresults/14/archives/36/langtools-36-summary.txt -O ${INPUT_FILE}`